{
    "id": "CVE-2019-5010",
    "cisaKEV": false,
    "reported_exploited": false,
    "exploit_maturity": "poc",
    "counts": {
        "public_exploit_count": 1
    },
    "timeline": {
        "nvd_published": "2019-10-31"
    },
    "exploits": [
        {
            "url": "https://github.com/JonathanWilbur/CVE-2019-5010",
            "name": "CVE-2019-5010 Exploit PoC - Python Denial of Service via Malformed X.509v3 Extension",
            "source": "github",
            "date_added": "2019-11-06",
            "exploit_maturity": "poc"
        }
    ]
}